Tasks/InstallSSHKey/installsshkey-util.ts

```diff
@@ -24,18 +24,9 @@ function execSshAddPassphraseSync(tool, args, passphrase):Q.Promise<boolean> {
     var defer = Q.defer<boolean>();
     let success = true;
 
-    let options:trm.IExecOptions = <trm.IExecOptions> {
-        cwd: process.cwd(),
-        env: process.env,
-        silent: false,
-        failOnStdErr: false,
-        ignoreReturnCode: true,
-        windowsVerbatimArguments: false,
-        outStream:<stream.Writable>process.stdout,
-        errStream:<stream.Writable>process.stderr
-    };
-
-    let cp = child.spawn(tool, args, {});
+    let cp = child.spawn(tool, args, {
```

Comment:
Based on the docs here: https://nodejs.org/api/child_process.html#child_process_options_detached, it seems that detached: true changes the default behavior of child process on windows (i.e. child process can run after parent is killed if this is set to true). Wouldn't it be better to check for OS version and pass detached: true only for Mac and Linux?